app/helpers/rademade_admin/field_helper.rb

```diff
@@ -35,8 +39,10 @@ def pagination_option(number, name = 'paginate')
   end
 
   def input_attr(attrs = {})
-    attrs.deep_merge :wrapper_html => { :class => 'form-group' },
-                :input_html => { :class => 'form-input' }
+    attrs.deep_merge(
+      wrapper_html: { :class => 'form-group' },
+      input_html: { :class => 'form-input' }
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.